packages/devtools/src/composite.ts

```diff
@@ -37,7 +38,8 @@ function isSignedCommit(input: Record<string, any>): input is SignedCommit {
 function toStrictDefinition(definition: InternalCompositeDefinition): StrictCompositeDefinition {
   const emptyViews = { account: {}, root: {}, models: {} }
   const views = definition.views ? { ...emptyViews, ...definition.views } : emptyViews
-  return { aliases: {}, commonEmbeds: [], ...definition, views }
+  const indices = definition.indices ? definition.indices : {}
```

Comment:
is `indices` an object or an array?  Should the default be `[]` instead of `{}`?

Reply:
It's a record.